src/graphql/queries/characterQueries.js

```diff
@@ -40,3 +40,25 @@ export const GET_ALL_CHARACTERS = gql`
     }
   }
 `
+
+export const GET_A_CHARACTER = gql`
```

Comment:
Careful with the naming here, this should be something like `CHARACTER_FRAGMENT`